packages/react-components/src/components/OMBInfo/OMBInfo.jsx

```diff
@@ -97,7 +97,7 @@ class OMBInfo extends React.Component {
           contents={
             this.props.children
               ? this.props.children
-              : this.modalContents(resBurden)
+              : this.modalContents(resBurden, hideResBurdenSection)
```

Comment:
`resBurden` is the minutes arg, so unless that prop is set and as long as we don't default it to something truthy, I think this might do what we want.